app/src/main/java/com/example/filemanager/AllFilesFragment.kt

```diff
@@ -6,16 +6,19 @@ import android.os.Environment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.Toast
 import androidx.annotation.RequiresApi
 import androidx.fragment.app.Fragment
-import androidx.recyclerview.widget.DefaultItemAnimator
+import androidx.recyclerview.widget.DividerItemDecoration
 import androidx.recyclerview.widget.LinearLayoutManager
-import com.example.filemanager.databinding.FragmentAllFilesBinding
+import androidx.recyclerview.widget.RecyclerView
 import java.io.File
+import java.util.ArrayList
+import com.example.filemanager.databinding.FragmentAllFilesBinding
 
-class AllFilesFragment : Fragment() {
+class AllFilesFragment : Fragment(), ItemClickListener {
     private lateinit var adapter: ListAdapter
-    private var _binding: FragmentAllFilesBinding? = null
+    private var _binding: FragmentAllFilesBinding ? = null
     private val binding get() = _binding!!
```

Comment:
why is it needed to use two variables for binding?

Reply:
I deleted _binding variable and I push the modifications into previous commit